From bbe18e7f6a7817a5ae088035688f42fd4be0bc38 Mon Sep 17 00:00:00 2001
From: Piotr Karbowski <piotr.karbowski@gmail.com>
Date: Wed, 11 Jan 2012 00:58:08 +0100
Subject: [PATCH] Disable DO_HOST_LOOKUP, don't query DNS. That should allow
 people log into initramfs without horrible lag when there
 is no nameservers or unable to connect to them.

---
 options.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/options.h b/options.h
index 73689ad..2c6aea0 100644
--- a/options.h
+++ b/options.h
@@ -144,7 +144,7 @@ much traffic. */
 #endif
 
 /* Whether to do reverse DNS lookups. */
-#define DO_HOST_LOOKUP
+/* #define DO_HOST_LOOKUP */
 
 /* Whether to print the message of the day (MOTD). This doesn't add much code
  * size */
-- 
1.7.8.3

